src/main/java/net/jodah/failsafe/function/BiPredicate.java

```diff
@@ -19,4 +19,5 @@
 @Deprecated
 public interface BiPredicate<T, U> {
   boolean test(T t, U u);
+  void dontAllowLambdaShorthand();
```

Comment:
This will change how things compile, which is what we want, but what does it do for binary compatibility? Will code that previously compiled against this using lambdas continue to run, or will we see exceptions?